integrated_channel_plugin_sapsf/models.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from model_utils.models import TimeStampedModel, TimeFramedModel
+
+
+# Create your models here.
+
+class OCNWebServicesConfiguration(TimeStampedModel):
+    """
+    The Global configuration for integrating with SuccessFactors.
+    """
+
+    completion_status_api_path = models.CharField(max_length=255)
+    course_api_path = models.CharField(max_length=255)
+    oauth_api_path = models.CharField(max_length=255)
+    provider_id = models.CharField(max_length=100, default='EDX')
+
+    class Meta:
+        app_label = 'integrated_channel_plugin_sapsf'
+
+
+class OCNWebServicesEnterpriseCustomerConfiguration(TimeStampedModel):
+    """
+    The Enterprise specific configuration we need for integrating with SuccessFactors.
+    """
+
+    enterprise_customer_uuid = models.UUIDField(unique=True)
```

Comment:
That depends on whether or not we would ever want to use this package without being tied to the concept of an enterprise. In my mind that doesn't seem likely, especially because our definition of an enterprise is pretty broad.
I guess we could put this new identifier in the plugin configuration model in enterprise that references the id for this model, is that what you would envision?